grafana/grafana
# InfluxDB: new row value types

**What this PR does / why we need it**:
Part of https://github.com/grafana/grafana/issues/43076

This PR contains the changes needed to parse queries that return results with string & boolean values (in the backend).

**Test scenarios** (via Explore):

1. Create a query that returns string/boolean values e.g. select * from logs where time >= now() - 30m and time <= now()
2. Be careful with the `- 30m` above as setting a value too big will potentially return tens of thousands of results. Maybe a smaller time like `- 10m` to start and increment until you get a number of results you are happy with.
3. Ensure the query renders in the table correctly. We could also check the HTTP response here to check all values are populated and none a nil (which was previously the case).

**Special notes for your reviewer**:
This worked before because we were querying through the proxy not the backend.
